Doc/library/test.rst

```diff
@@ -260,7 +280,7 @@ The :mod:`test.support` module defines the following constants:
 
 .. data:: TESTFN_UNICODE
 
-    Set to a non-ASCII name for a temporary file.
+    Set to a non-ASCII filename containing several unicode characters.
```

Comment:
Looks redundent to me (*non-ASCII* and *several unicode*). Also an "unicode character" may be one in the ASCII table.

Maybe "Set to a non-ASCII filename." is enough? In contrast this one is *not* tested against being fsencodable.